kubetest2/kubetest2-gke/deployer/commandutils.go

```diff
@@ -29,6 +29,22 @@ func (d *deployer) prepareGcpIfNeeded() error {
 	// TODO(RonWeber): This is an almost direct copy/paste from kubetest's prepareGcp()
 	// It badly needs refactored.
 
+	var endpoint string
+	switch env := d.environment; {
```

Comment:
nit: use env as the switch condition so that all cases are simplified
```
switch env := d.environment; env {
case "test": ...
}
```

Reply:
can't switch on `env` string because the last case is matching on a regex `urlRe.MatchString(env)` and the result is a bool.

Reply:
ahh right